src/main/scala/io/iohk/ethereum/db/storage/ReferenceCountNodeStorage.scala

```diff
@@ -98,6 +115,16 @@ class ReferenceCountNodeStorage(nodeStorage: NodesStorage, blockNumber: Option[B
 
 object ReferenceCountNodeStorage extends PruneSupport with Logger {
 
+  val nodeKeyLength = 32
+
+  def drRowKey(bn: BigInt): ByteString = {
+    ByteString("dr".getBytes()) ++ ByteString(bn.toByteArray)
```

Comment:
Maybe deathRows could be stored in separate column family (separate cache, etc)?

Reply:
it is possibility, but i do not think it is worth it, there is only as many deathrows as history blocks in pruning, which currently is 64, and separate column for only 64 key value pairs sound like overkill.